roles/openshift_node/tasks/main.yml

```diff
@@ -36,7 +36,7 @@
 
 - name: Install sdn-ovs package
   action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-sdn-ovs{{ openshift_pkg_version | oo_image_tag_to_rpm_version(include_dash=True) }} state=present"
-  when: openshift.common.use_openshift_sdn and not openshift.common.is_containerized | bool
+  when: openshift.common.use_openshift_sdn and not openshift.common.use_flannel | bool and not openshift.common.is_containerized | bool
```

Comment:
@detiber Right, which is why we currently require setting both `openshift_use_openshift_sdn=false` and `openshift_use_flannel=true`. What if we approached this as an "sdn kind" sort of variable -- defaults to openshift, can be set to nuage, flannel or X?